Integrations/ProofpointTAP_v2/ProofpointTAP_v2_test.py

```diff
@@ -260,4 +269,13 @@ def test_next_fetch(requests_mock):
 
     assert len(incidents) == 4
     assert json.loads(incidents[0]['rawJSON'])["messageID"] == "1111@evil.zz"
-    assert next_run == {"last_fetch": "2010-01-30T00:01:00.000Z"}
+
+
+def test_get_fetch_times():
```

Comment:
add one more test with predefined `last_fetch="2019-01-01T22:00:00"` and validate the array first 3 objects.
expected to be 
`["2019-01-01T22:10:05", "2019-01-01T23:10:05", "2019-01-02T00:10:05", ...] `